scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -101,15 +103,21 @@ trait Config extends Serializable {
   def setMapSpillThreshold(count: Int): Config =
     this + (SpillableProps.MAP_THRESHOLD -> count.toString)
 
+  @deprecated("deprecated in Cascading 2.7 and dropped in Cascading 3.0, use setMapSideAggregationCapacity", "cascading 2.7")
+  def setMapSideAggregationThreshold(count: Int): Config = {
```

Comment:
+1 to what @ianoc said. I'd rather us take the max value of the two keys in Operations.scala, and here set the non-deprecated internally, but not deprecate our API (instead explain what it means to the user, not necessarily in cascading terms).